### ISSUE TITLE
EOFError while using pandarallel

### ISSUE THREAD

**Issue by reach4bawer** [nalepae/pandarallel#161]
I tried using this package to query API end points based on the pandas dataframe. I am getting the following error using the package -

```
Traceback (most recent call last):
  File "<input>", line 1, in <module>
  File "/Users/my_user/project/venv/lib/python3.9/site-packages/pandarallel/pandarallel.py", line 453, in closure
    results = get_workers_result(
  File "/Users/my_user/project/venv/lib/python3.9/site-packages/pandarallel/pandarallel.py", line 366, in get_workers_result
    message_type, message = queue.get()
  File "<string>", line 2, in get
  File "/usr/local/Cellar/python@3.9/3.9.7/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/managers.py", line 810, in _callmethod
    kind, result = conn.recv()
  File "/usr/local/Cellar/python@3.9/3.9.7/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/connection.py", line 255, in recv
    buf = self._recv_bytes()
  File "/usr/local/Cellar/python@3.9/3.9.7/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/connection.py", line 419, in _recv_bytes
    buf = self._recv(4)
  File "/usr/local/Cellar/python@3.9/3.9.7/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/connection.py", line 388, in _recv
    raise EOFError
EOFError
```

**Comment by sacharbit** [comment]
I've got the same issue! I can't seem to find a solution.

Did you solve it?

**Comment by till-m** [comment]
Closed in favour of #178.
